File: ejercicio2.py
n=int(input("Cantidad de casos a evaluar: "))
i=0
while (i<n):    #ciclo que se repite segun cantidad de casos asignados
    i=i+1
    print("Caso ",i,": ")
    a=int(input("Primer numero: ")) #solicitud de valores
    b=int(input("Segundo numero: "))
    if a<b:     #menor
        print(a," es menor que ",b) 
    if a>b:     #mayor
        print(a," es mayor que ",b)
    if a==b:    #igual
        print(a," es igual a ",b)
        

File: ejercicio2.cpp
#include <iostream>
using namespace std;

int main(){
    //solicitud de cantidad de casos
    int n=0;
    cout << "Digite numero de casos "<<endl;cin >> n;
    //processo que se repite segun la cantidad de casos asignados
    for (int i = 1;i<=n;i++){
        
        int a=0;
        int b=0;
        //solicitud de valores
        cout << "Caso " << i << ": "<<endl;
        cout << "Primer numero:"<<endl; cin >> a;
        cout << "Segundo numero:"<<endl; cin >> b;
        //menor
        if (a<b){
            cout << a << " es menor a " << b << endl;
        }
        //mayor
        if (a>b){
            cout << a << " es mayor a " << b << endl;
        }
        //igual
        if (a==b){
            cout << a << " es igual a " << b << endl;
        }

    }
return 0;
}